api/src/test/java/org/openmrs/module/openconceptlab/updater/ImporterTest.java

```diff
@@ -671,6 +716,77 @@ public void importMapping_addConceptMappingAndUnretireTerm() throws Exception {
 		ConceptMapType mapType = conceptService.getConceptMapTypeByName("SAME_AS");
 		assertThat(concept.getConceptMappings(), contains(hasMapping(source, "1001", mapType)));
 	}
+	
+	@Test
+	public void importMapping_shouldUpdateMappingOnylIfItHasBeenUpdatedSinceLastImport() throws Exception {

```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Method names should not contain underscores](https://www.codacy.com/app/agrzybkowski/openmrs-module-openconceptlab/file/2377174879/issues/source?bid=3218637&fileBranchId=3224807#l721)